clevertap-core/src/test/java/com/clevertap/android/sdk/network/NetworkManagerTest.kt

```diff
@@ -163,8 +165,8 @@ class NetworkManagerTest : BaseTestCase() {
         val controllerManager =
             ControllerManager(appCtx, cleverTapInstanceConfig, lockManager, callbackManager, deviceInfo, dbManager)
         val cryptHandler = CryptHandler(0, AES, cleverTapInstanceConfig.accountId)
-        val localDataStore =
-            LocalDataStoreProvider.provideLocalDataStore(appCtx, cleverTapInstanceConfig, cryptHandler, deviceInfo)
+       /* val localDataStore =
```

Comment:
Why do we have this commented out here ?

Reply:
it's unused code, removed here https://github.com/CleverTap/clevertap-android-sdk/commit/2da2fed568849c7b4b555f8f5495dd9c14d5fffd